src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -70,49 +61,10 @@ def validate(self, attrs: Mapping[str, Any]) -> Mapping[str, Any]:
         if not attrs.get("raw", "").strip():
             return attrs
 
-        external_association_err: List[str] = []
-        # Get list of team/user names from CODEOWNERS file
-        team_names, usernames, emails = parse_code_owners(attrs["raw"])
-
-        # Check if there exists Sentry users with the emails listed in CODEOWNERS
-        user_emails = UserEmail.objects.filter(
-            email__in=emails,
-            user__sentry_orgmember_set__organization=self.context["project"].organization,
-        )
-
-        user_emails_diff = validate_association(emails, user_emails, "emails")
-        external_association_err.extend(user_emails_diff)
-
-        # Check if the usernames have an association
-        external_actors = ExternalActor.objects.filter(
-            external_name__in=usernames + team_names,
-            organization=self.context["project"].organization,
+        associations, _ = ProjectCodeOwners.validate_codeowners_associations(
```

Comment:
You should leave a comment in the serializer explaining why you're discarding `errors`.